src/Microsoft.DotNet.XHarness.CLI/CommandArguments/WASM/Arguments/LocaleArgument.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.DotNet.XHarness.CLI.CommandArguments.Wasm;
+using System;
+
+internal class LocaleArgument : RequiredStringArgument
+{
+    public LocaleArgument(string defaultValue)
+        : base("locale=", $"Sets LANG environment variable, default value {defaultValue}", defaultValue)
```

Comment:
nit: I guess this text doesn't apply for node so we could make it more generic and not spell out the concrete variable?

E.g. 
```suggestion
        : base("locale=", $"Sets the browser's/node's language through an environment variable, default value {defaultValue}", defaultValue)
```

Reply:
Also it's clear it's not setting XHarness' output language or similar